scripts/Microsoft.DotNet.Cli.Build.Framework/TargetConditions/EnvironmentAttribute.cs

```diff
@@ -24,15 +23,22 @@ public override bool EvaluateCondition()
         {
             var actualVal = Environment.GetEnvironmentVariable(_envVar);
 
-            foreach (var expectedVal in _expectedVals)
+            if (_expectedVals.Length == 0)
```

Comment:
You can throw an `ArgumentNullException` here - https://github.com/dotnet/cli/pull/1824/files#diff-ec153fc2353f931636f6e56f3b335449R11